RecoBTag/DeepFlavour/interface/c_pf_features_converter.h

```diff
@@ -0,0 +1,263 @@
+#ifndef RecoSV_DeepFlavour_c_pf_features_converter_h
+#define RecoSV_DeepFlavour_c_pf_features_converter_h
+
+#include "RecoBTag/DeepFlavour/interface/deep_helpers.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+
+namespace deep {
+
+  // conversion map from quality flags used in PV association and miniAOD one
+  constexpr int qualityMap[8]  = {1,0,1,1,4,4,5,6};
+
+  enum qualityFlagsShiftsAndMasks {
+        assignmentQualityMask = 0x7, assignmentQualityShift = 0,
+        trackHighPurityMask  = 0x8, trackHighPurityShift=3,
+        lostInnerHitsMask = 0x30, lostInnerHitsShift=4,
+        muonFlagsMask = 0x0600, muonFlagsShift=9
+  };
+
+
+  template <typename CandidateType, typename JetType,
+            typename TrackInfoBuilderType,
+            typename ChargedCandidateFeaturesType>
+  void c_pf_packed_features_converter(const CandidateType * c_pf,
```

Comment:
```ChargedCandidateToFeatures``` converter may look better when this is moved to camel caps

Reply:
Yes

Reply:
change in https://github.com/cms-sw/cmssw/pull/19893/commits/c58a7db3f5b57532c429a7f2f102b462285b57a4